### ISSUE TITLE
Version Flag states "nothing"

### ISSUE THREAD

**Issue by JeffreyMaurer** [kharchenkolab/Baysor#115]
I `wget https://github.com/kharchenkolab/Baysor/releases/download/v0.6.2/baysor-x86_x64-linux-v0.6.2_build.zip`, unzipped it and added it to my path. I can see the help.

When I use the `--version` flag, I get the literal text "nothing" instead of something along the lines of "Baysor v0.6.2".

![version_nothing_baysor](https://github.com/kharchenkolab/Baysor/assets/29780865/4decf463-50d3-4190-81b2-4f1259607c92)

**Comment by VPetukhov** [comment]
Good catch! :) Unfortunately, it's not my function, but something from the package building system, so I don't have much control to fix it. As it's not critical, I'll just hope that it will be fixed on their side.

**Comment by VPetukhov** [comment]
Now, this is fixed in v0.7.0
